_posts/2016-12-05-moore-report.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+subheadline: "Support"
+title: "A Year in Review: Annual Moore Progress Report"
+teaser: ""
+header:
+  image_fullwidth: "light-blue-wood-texture.jpg"
+categories:
+  - blog
+comments: true
+show_meta: true
+authors: ["Tracy Teal"]
+---
+
+
+A year ago we were fortunate enough to receive a grant from the 
+[Gordon and Betty Moore Foundation](https://www.moore.org/article-detail?newsUrlName=the-gordon-and-betty-moore-foundation-selects-awardees-for-$21-million-in-grants-to-stimulate-data-driven-discovery) 
+Data-Driven Discovery initiative to support Data Carpentry to develop and deliver data skills training to researchers in the life and 
+physical sciences. [See our original proposal here.](https://figshare.com/articles/Data_Carpentry_Moore_Foundation_Grant_Proposal/1507552)
+Thanks to the Moore Foundation, this support has allowed us to run more workshops, develop new materials, 
+develop infrastructure and advocate for the importance of data skills training for more effective and reproducible research.  
+
+
+We wanted to share the outputs and outcomes of Data Carpentry activities over this last year from our annual report to the Moore 
+Foundation.  
+
```

Comment:
[suggestion] The work of the community has made this possible, and we wanted to share our Moore annual report to highlight all the work that has been done in the last year!